frontend/pages/hosts/ManageHostsPage/ManageHostsPage.jsx

```diff
@@ -199,12 +190,6 @@ export class ManageHostsPage extends PureComponent {
     }
   }
 
-  filterAllHosts = (hosts, selectedLabel) => {
```

Comment:
This was weird. `filterAllHosts` was only ever used in the tests! I was able to remove this and then rewrite the test to make it actually useful.